opal/mca/base/mca_base_var.c

```diff
@@ -113,6 +128,11 @@ static const char *info_lvl_strings[] = {"user/basic",  "user/detail",  "user/al
                                          "tuner/basic", "tuner/detail", "tuner/all",
                                          "dev/basic",   "dev/detail",   "dev/all"};
 
+static char *var_dump_color_key_names[VAR_DUMP_COLOR_KEY_COUNT] = {
+    "name", "value", "valid_values"
+};
```

Comment:
```suggestion
static char *var_dump_color_key_names[VAR_DUMP_COLOR_KEY_COUNT] = {
    [VAR_DUMP_COLOR_VAR_NAME] = "name",
    [VAR_DUMP_COLOR_VAR_VALUE] = "value",
    [VAR_DUMP_COLOR_VALID_VALUES] = "valid_values",
};
```

Using the C99 array initializer syntax guarantees that the strings match the correct index.

Reply:
interesting